src/www/services_dhcp_edit.php

```diff
@@ -519,7 +519,7 @@ function show_tftp_config() {
 									</div>
 									<div id="showtftp" style="display:none">
 										<input name="tftp" type="text" class="formfld unknown" id="tftp" size="50" value="<?=htmlspecialchars($pconfig['tftp']);?>" /><br />
-										<?=gettext("Leave blank to disable.  Enter a full hostname or IP for the TFTP server.");?>
+										<?=gettext("Leave blank to disable. Enter a full hostname or IP for the TFTP server.");?>
```

Comment:
Small note on these ".  " type things: it's not a mistake, this is old typewriter end of sentence style still employed in technical BSD manuals. But I agree that we should migrate them bit by bit to what you've done here and elsewhere. thanks :)